ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Analysis.hs

```diff
@@ -178,15 +179,19 @@ data TraceEvent blk =
     --   * slot number when the block was forged
     --   * number of transactions in the block
     --   * total size of transactions in the block
-  | BlockMempoolAndForgeRepro BlockNo SlotNo Int SizeInBytes IOLike.DiffTime IOLike.DiffTime
-    -- ^ triggered for all blocks during ShowBlockTxsSize analysis,
+  | BlockMempoolAndForgeRepro BlockNo SlotNo Int SizeInBytes IOLike.DiffTime Int64 Int64 IOLike.DiffTime Int64 Int64
+    -- ^ triggered for all blocks during MempoolAndForgeRepro analysis,
     --   it holds:
     --   * block number
     --   * slot number when the block was forged
     --   * number of transactions in the block
     --   * total size of transactions in the block
     --   * time to tick ledger state
+    --   * mutator time to tick ledger state
```

Comment:
Perhaps we should say

```
Total time spent in the mutator when ticking the ledger state
```

https://github.com/input-output-hk/ouroboros-consensus/pull/400/files#diff-fa79e075b3c0bc1a549ac7ddfe0b97fc5817a24a8de7577988d15d92019316efR161

Reply:
Done